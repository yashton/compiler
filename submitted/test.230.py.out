(ID "a")
(PUNCT "=")
(ERROR "Illegal character '\   
 b' on line 2)
