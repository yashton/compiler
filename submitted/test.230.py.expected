(ID "a")
(PUNCT "=")
(ERROR "unknown char: '\'")
(NEWLINE)
(INDENT)
(ID "b")
(DEDENT)
(ENDMARKER)
